.github/workflows/wheels.yml

```diff
@@ -0,0 +1,74 @@
+name: wheels
+
+on:
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Name'
+        required: false
+        default: 'No name specified'
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+        - os: ubuntu-20.04
+          CIBW_ENVIRONMENT: "CXX=$(which g++) OPEN_SPIEL_BUILDING_WHEEL='ON' OPEN_SPIEL_BUILD_WITH_ACPC='ON' OPEN_SPIEL_BUILD_WITH_HANABI='ON'"
+        - os: macOS-10.15
+          CIBW_ENVIRONMENT: "OPEN_SPIEL_BUILDING_WHEEL='ON' OPEN_SPIEL_BUILD_WITH_ACPC='ON' OPEN_SPIEL_BUILD_WITH_HANABI='ON'"
+    env:
+      OPEN_SPIEL_BUILDING_WHEEL: ON
+      OPEN_SPIEL_BUILD_WITH_ACPC: ON
+      OPEN_SPIEL_BUILD_WITH_HANABI: ON
+      OS_PYTHON_VERSION: 3.9
+      # While testing, try specific versions
+      CIBW_BUILD: cp36-* cp37-* cp38-* cp39-*
+      CIBW_PROJECT_REQUIRES_PYTHON: ">=3.6"
+      CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+      CIBW_MANYLINUX_I686_IMAGE: manylinux2014
+      CIBW_SKIP: cp27-* pp*
+      CIBW_BEFORE_BUILD: python -m pip install --upgrade cmake
+      CIBW_ENVIRONMENT: ${{ matrix.CIBW_ENVIRONMENT }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Install
+        run: |
+          pwd
+          uname -a
+          which python
+          which g++
+          g++ --version
+          python --version
+          chmod +x install.sh
+          ./install.sh
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools
+          python -m pip install --upgrade -r requirements.txt -q
+          source ./open_spiel/scripts/python_extra_deps.sh
+          python -m pip install --upgrade $OPEN_SPIEL_PYTHON_JAX_DEPS $OPEN_SPIEL_PYTHON_PYTORCH_DEPS $OPEN_SPIEL_PYTHON_TENSORFLOW_DEPS $OPEN_SPIEL_PYTHON_MISC_DEPS
+          python -m pip install twine build
+          python -m pip install cibuildwheel==1.10.0
+      - name: Build sdist
+        run: |
+          python -m build -s
+          twine check dist/*.tar.gz
+          
+      - name: Build bdist_wheel
+        run: python -m cibuildwheel --output-dir wheelhouse
+
+      - name: Install and test bdist_wheel
+        run: ./open_spiel/scripts/test_wheel.sh
```

Comment:
FYI, you can tell cibuildwheel to do the testing via variables too; if you do that, you can remove most of the install step.

Reply:
Thanks for the suggestions!

Do you mean it can do the testing from within the same virtual environments that have built the wheels? That would be ideal, yes! Do you have an example you can point me to on how to run arbitrary commands within them?

Reply:
https://cibuildwheel.readthedocs.io/en/stable/options/#test-command - multiple commands should be joined with && on all platforms.

Reply:
That's awesome, thanks for the pointer! I will take a look once I figure out how to fix the current build issue.

Reply:
FYI, cibuildwheel 2.0 will add a config mode which is really nice for adding things like this (see https://github.com/scikit-build/cmake-python-distributions/pull/139/commits/bc80ddb3bc13f1d28d9a1e28b36ba9ce6f92cba9 for a great example of it). 2.0.0a4 is fully usable as long as you don't activate the new "build" backend instead of pip (ironically broken due to a `pypa/pip` bug, not a `pypa/build` bug).

Reply:
Ok, thanks again for all the great feedback. 

I have now run a basic test suite within the Docker image by the `CIBW_TEST_COMMAND` and larger tests externally on just the two platforms. 

Our release has been delayed for some time waiting only on this bdist wheel, and there are a few other reasons that 
I need to push it forward. Unfortunately, I won't have the time to address everything since I have quite limited time to spend on this. So some of these points will have to be deferred (most notably a move to the TOML -- I am unable to invest the time into it right now). I appreciate the help nonetheless, and will look into it for future releases.